### PR TITLE
Component submission docs

### PR DESCRIPTION
**Pull request by shinytoyrobots**
Updated the url so that it doesn't clash with the rewrite.config from #1631 .

### CODE REVIEW

**Review by andreancardona — APPROVED** [review]
@shinytoyrobots looks good!

File: src/pages/community/contribute-a-component.mdx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+   design_asset: choose one of Sketch | Axure | Adobe XD | Figma
+   platform: Web
+   aliases:
+     - fuzzy term for component (e.g. an alias for a "card" component might be "tile")
+   date_added: YYY-MM-DDT00:00:00.000Z
+   ```
+
+5. Once done, click the green `Commit new file` button at the bottom.
+
+After submitting the PR for the data, you should upload a thumbnail image. Go
```
Comment (janchild) [inline src/pages/community/contribute-a-component.mdx:48]:
```suggestion
After submitting the PR for the data, you should upload a thumbnail image. 

1. Go
```

File: src/pages/community/contribute-a-component.mdx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+back to your team folder in the
+[index](https://github.com/carbon-design-system/carbon-website/tree/master/src/data/index).
+
```
Comment (janchild) [inline src/pages/community/contribute-a-component.mdx:51]:
```suggestion
```

**Review by janchild — COMMENTED** [review]
minor fix

**Review by janchild — APPROVED** [review]

**Pull request merged**